src/web/src/components/EmployeeSearch.vue

```diff
@@ -11,8 +11,8 @@
                     </v-breadcrumbs-item>
                 </template>
             </v-breadcrumbs>
-
-            <h2 v-if="results" class="px-0" style="font-size: 34px !important;">There are no results</h2>
+            <h2 v-if="results && department !== 'Any department'" class="px-0" style="font-size: 34px !important;">Your search for {{ this.searchTitle.replace(/-/g, " ") }} in the dpto {{this.department}} had no results.</h2>
```

Comment:
Is `the dpto` correct here? Looks like it might be a typo.

Reply:
Thanks @leeomara , yes, it was a typo, and I have updated it.